### PR TITLE
added HTML widget

### LINKED ISSUE #2861: Feature request - HTML viewer widget
Feature Request : 
Current scenario - There is no widget, which will show an HTML page/ window upon providing raw HTML data

What we can do to add the above functionality - 
We can add a feature where users can provide raw HTML and can see the output in the application itself using the HTML viewer widget

### PR DESCRIPTION
**Pull request by manishkushare**
Resolve #2861

### CODE REVIEW

**Comment by sherfin94** [comment]
@manishkushare You seem to have pushed package-lock.json without having installed any new packages. Please remove these changes.

image

File: frontend/src/Editor/Components/components.js [hunk 1]
```diff
@@ -2021,4 +2021,43 @@ export const componentTypes = [
...
+      value: {},
+    },
+    definition: {
+      others: {
+        showOnDesktop: { value: '{{true}}' },
+        showOnMobile: { value: '{{false}}' },
+      },
+      properties: {
+        rawHtml: {
+          value: '',
```
Comment (sherfin94) [inline frontend/src/Editor/Components/components.js:2054]: Lets set some simple sample HTML as default value here.

**Comment by sherfin94** [comment]
Is it possible to also add another property called `CSS` , where we can supply css that will be applied to the HTML that is rendered?

**Review by sherfin94 — CHANGES_REQUESTED** [review]
I have requested some changes. Additionally, please also add documentation and resolve conflicts with `develop` branch.

Widget works well by the way 😄

**Comment by manishkushare** [comment]
Sure, on it

**Pull request closed without merging**